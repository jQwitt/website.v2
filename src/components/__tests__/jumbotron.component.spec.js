import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Jumbotron } from '../index';

afterEach(cleanup);
describe('Jumbotron renders correctly', () => {
  test('snapshot', () => {
    //const { asFragment } = render(<Jumbotron />);
    //expect(asFragment()).toMatchSnapshot();
    expect(1).toBe(1);
  });
});
