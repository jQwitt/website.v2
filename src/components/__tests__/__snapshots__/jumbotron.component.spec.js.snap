// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron renders correctly snapshot 1`] = `
<DocumentFragment>
  <div
    class="col-12 row full-viewport content-jumbo"
  >
    <div
      class="mx-auto main"
    >
      <svg />
    </div>
    <div
      class="col-12 text-center center-bottom"
    >
      <i
        class="flaticon-down-arrow large pulse"
      />
    </div>
  </div>
</DocumentFragment>
`;
